File: Trivia/GetRoomsResponse.cpp
#include "GetRoomsResponse.h"

GetRoomsResponse::GetRoomsResponse(unsigned int _status, vector<Room> _rooms) : status(_status), rooms(_rooms)
{
}

vector<Room> GetRoomsResponse::getRooms()
{
	return rooms;
}

unsigned int GetRoomsResponse::getStatus()
{
	return status;
}

File: TriviaClient/Responses/GetRoomsResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TriviaClient.infrastructure;

namespace TriviaClient.Responses
{
    class GetRoomsResponse
    {
        private int status;
        private Room[] rooms;
        public GetRoomsResponse(Room[] rooms, int status)
        {
            this.rooms = rooms;
            this.status = status;
        }

        public Room[] GetRooms()
        {
            return rooms;
        }

        public int GetStatus()
        {
            return status;
        }
    }
}
